app/Http/Controllers/StatusPageController.php

```diff
@@ -209,4 +209,36 @@ public function showComponentBadge(Component $component)
 
         return Response::make($badge, 200, ['Content-Type' => 'image/svg+xml']);
     }
+
+    /**
+     * Show the privacy statement.
+     *
+     * @return \Illuminate\Routing\Redirector|\Illuminate\Http\RedirectResponse|\Illuminate\View\View
+     */
+    public function showPrivacyStatement()
+    {
+        $privacyStatement = trim(Config::get('setting.privacy_statement', ''));
+        if (starts_with($privacyStatement, 'https://') && filter_var($privacyStatement, FILTER_VALIDATE_URL)) {
+            return redirect($privacyStatement);
```

Comment:
Can we avoid using this helper please.

Reply:
Sure. Only out of interest: Is using the helper functions considered "bad practice"? Some projects seem to prefer them over using the class methods.

Reply:
My issue with helpers is because of things like this: https://laravel-news.com/laravel-5-8-deprecates-string-and-array-helpers